tests/functional/api.php

```diff
@@ -21,6 +21,7 @@
     ['v1/suggestions/central/en-US/fr/bookmark/?max_results=2', 200, '["Bookmark","Bookmarks"]'],
     ['v1/tm/central/en-US/fr/Bookmark/?max_results=3&min_quality=80', 200, '[{"source":"Bookmark","target":"Marquer cette page","quality":100},{"source":"Bookmark","target":"Marque-page","quality":100},{"source":"Bookmarks","target":"Marque-pages","quality":88.89}]'],
     ['v1/tm/global/fr/en-US/Ouvrir/', 200, '[{"source":"Ouvrir dans le Finder","target":"Find in Finder","quality":28.57},{"source":"D\u00e9couvrez comment ouvrir une fen\u00eatre de navigation priv\u00e9e","target":"Learn how to open a private window","quality":8.77}]'],
+    ['v1/transliterate/serbian/%D0%A1%D1%80%D0%B1%D0%B8%D1%98a/', 200, '{"transliterated_string":"SrbiJa"}'],
```

Comment:
We can also check this one:
['v1/transliterate/foo/bar/', 400, '{"error":"Wrong language"}'],

Also, current test is failing on my Ubuntu VM:
* Expected: {"transliterated_string":"SrbiJa"}
* Received: {"transliterated_string":"Srbija"}

Probably related to ICU version, not sure we can fix that :-/
I’m using php5-intl_5.6.11+dfsg-1ubuntu3.1_amd64.deb